### ISSUE TITLE
Make plugin more keyboard friendly

### ISSUE THREAD

**Issue by shbach** [Retr0ve/joplin-plugin-emoji#1]
Already briefly mentioned [here](https://discourse.joplinapp.org/t/plugin-emoji-picker/24228/8?u=shbach) but creating an issue to further elaborate and for better tracking.

Right now the plugin isn't the most keyboard friendly. When the pop up is activated I have to then use my mouse to select the desired emoji. Ideally I can just type until I have a match then press enter without having to leave the keyboard. This means the first result in the list has to be selected, arrows can be used to navigate, and enter/tab could be used to insert.

I think looking at the way others do it such as Discourse and Github could be a good reference.

**Comment by andypiper** [comment]
Yes, exactly this - I just installed the plugin and had the same thought (I'm on a Mac). It would help if I could quickly navigate and hit enter to select the emoji.
